ecosystem/typescript/sdk/src/plugins/coin_client.ts

```diff
@@ -129,10 +130,11 @@ export class CoinClient {
       // The coin type to use, defaults to 0x1::aptos_coin::AptosCoin.
       // If you want to check the balance of a fungible asset, set this param to be the
       // fungible asset address
-      coinType?: string;
+      coinType?: string | MaybeHexString;
```

Comment:
```suggestion
      coinType?: MaybeHexString;
```

since `type MaybeHexString = HexString | string | HexEncodedBytes;`

Reply:
oh right, I'll change the other one then

Reply:
well actually, I think the string is indicating it can be a regular `TypeTag` string as well. Does it make sense to make it a `MaybeHexString` if it's not intended to be used as a `HexString`?

Reply:
I'm going to change it back (to the new change, aka `string | MaybeHexString`) to make it clearer that you can use either a TypeTag string or a (Maybe)HexString address